### PR TITLE
Feature/add header

### PR DESCRIPTION
**Pull request by abdelhamidaitayoub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <div>dfsdf</div>
```
Comment (abdelhamidaitayoub) [inline index.html:11]: jjhjhadjhad

File: index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (touzaelhassan) [inline index.html:3]: vgc

Comment (touzaelhassan) [inline index.html:3]: vv

**Review by touzaelhassan — APPROVED** [review]
Everything  is ok

**Pull request merged**